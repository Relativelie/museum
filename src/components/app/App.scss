@import "../../variables";

body {
    background-color: $bgDark;
}
