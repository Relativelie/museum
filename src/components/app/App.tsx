import { Carousel } from "../carousel/Carousel";
import { Menu } from "../menu/Menu";
import "./App.scss";

function App() {
  return (
    <div className="App">
      <Menu />
      <Carousel/>
    </div>
  );
}

export default App;
