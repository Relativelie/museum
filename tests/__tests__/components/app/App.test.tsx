import '@testing-library/jest-dom/extend-expect';
import '@testing-library/jest-dom';
import * as ShallowRenderer from 'react-test-renderer/shallow';
import App from '../../../../src/components/app/App';

describe('App component', () => {
    test('snapshot - app component', () => {
        const renderer = ShallowRenderer.createRenderer();
        renderer.render(<App />);
        const tree = renderer.getRenderOutput();
        expect(tree).toMatchSnapshot();
    });
});
