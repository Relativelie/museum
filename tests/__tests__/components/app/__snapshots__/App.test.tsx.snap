// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback - App component snapshot - app component 1`] = `
<div
  className="App"
>
  <Menu />
  <CarouselComponent />
  <Faq />
  <News />
  <Feedback />
  <Footer />
</div>
`;
